[ADD] not existing article parts

JochenPfaeffle opened the pull request:
Add not existing components. This Components exist in BMEcat v1.2.
Add Article Keywords.
Add Article Mime Types for Documents and images.
Add Article Order Details.
Add Article Status.
Add Buyer Aid.
Add Special Treatment Class.

Commit:
[ADD] not existing article parts

sveneisenschmidt commented:
@JochenPfaeffle Thanks for the Pull Request. Can you please check why the test/build job is failing?

Thank you

JochenPfaeffle commented:
I do this at the moment

JochenPfaeffle commented:
At the moment I have only this error:
SE\Component\BMEcat\Tests\Node\ArticleNodeTest::Serialize_Without_Null_Values
JMS\Parser\SyntaxErrorException: Expected T_NAME, but got "\" of type T_NONE at beginning of input.
but I don't now what the problem is.

Commit:
[FIX] problems from test

Commit:
[FIX] problems from test

JochenPfaeffle commented:
Now the test is success.

Commit:
[UPDATE] composer lock
[FIX] Problem of To_String_Returns_Default_Document_Without_Null_Values with the new versions

Commit:
[RESET] reset composer lock and empty_document_without_null_values to pre version

Commit:
[UPDATE] composer, empty_document_without_null_values and travis.yml

Commit:
[CHANGE] Remove the cdata parts with adding @Serializer\XmlElement(cdata=false) in the php-doc

Commit:
[ADD] Options to Feature

Commit:
[ADD] CDATA only on the string parameter

JochenPfaeffle commented:
It is fixed and needed parts added.

Commit:
[CHANGE] BMEcat tag to BMECAT

Commit:
[ADD] Needed article parts from BMEcat and for PIXI Imports

Commit:
[ADD] WEIGHT for PIXI Import

Commit:
[CHANGE] WEIGHT int to float

Commit:
[ADD] needed Parts to Catalog Header

sveneisenschmidt commented:
Hello @JochenPfaeffle,
Sorry for the late reply. 

As I am quite busy with a lot of topics next to maintaining some of these projects would you be open to take on co-maintaining this project?

Let me know what you think
Sven
